README.md

```diff
@@ -125,6 +152,20 @@ as discussed [here](https://github.com/projecthamster/hamster/pull/421#issuecomm
 
 Now restart your panels/docks and you should be able to add Hamster!
 
+##### Flatpak
+
+Alternatively, you can also build a sandboxed
```

Comment:
This suggests it just builds the flatpack, but I think it also installs it. Might be good to clarify.